### DISCUSSION TITLE
How to set scrollLeft

### DISCUSSION THREAD

**Discussion by Lowgy** [katspaugh/wavesurfer.js#3962] [Q&A] [answered]
I am trying to create a custom tool on top of WaveSurfer. The functionality I am going for is a custom "scrollbar", I essentially want to be able to scroll the audio timeline from outside wavesurfer. Is setting the WaveSurfer scrollLeft the correct way to handle this?

**Comment by katspaugh** [comment, marked as answer]
https://wavesurfer.xyz/docs/classes/wavesurfer.default#setScroll – in pixels
https://wavesurfer.xyz/docs/classes/wavesurfer.default#setScrollTime – in seconds
